arviz/stats/stats.py

```diff
@@ -1627,34 +1631,17 @@ def loo_pit(idata=None, *, y=None, y_hat=None, log_weights=None):
             f"{y_hat.shape,} and {log_weights.shape}"
         )
 
-    kwargs = {
-        "input_core_dims": [[], ["__sample__"], ["__sample__"]],
-        "output_core_dims": [[]],
-        "join": "left",
-    }
-    ufunc_kwargs = {"n_dims": 1}
-
+    if y.ndim == 2:
+        y = y.T[None, :]
+    y_hat = y_hat.T
     if y.dtype.kind == "i" or y_hat.dtype.kind == "i":
         y, y_hat = smooth_data(y, y_hat)
 
-    return _wrap_xarray_ufunc(
-        _loo_pit,
-        y,
-        y_hat,
-        log_weights,
-        ufunc_kwargs=ufunc_kwargs,
-        **kwargs,
-    )
-
+    mlw = np.max(log_weights)
+    pits = np.exp(mlw + np.log(np.sum(np.exp(log_weights.T - mlw) * (y_hat <= y), axis=0)))
```

Comment:
`np.log(np.sum(np.exp(...)))` isnt this the logsumexp?

Reply:
yes